src/pandas_profiling/config.py

```diff
@@ -1,9 +1,13 @@
 """Configuration for the package."""
 from enum import Enum
 from typing import Any, Dict, List, Optional
-
+import warnings
 from pydantic import BaseModel, BaseSettings, Field
 
+# Comment this function to see Warnings in console
+def warn(*args, **kwargs):
+    pass
+warnings.warn = warn
```

Comment:
A feature to disable warnings is desirable, but it should not be enabled/disabled by commenting code, also the default should be having the warnings enabled.